stl/inc/format

```diff
@@ -656,16 +667,16 @@ public:
         _Specs._Fill[0]   = _CharT{'0'};
     }
 
-    constexpr void _On_width(unsigned int _Width) {
+    constexpr void _On_width(int _Width) {
         _Specs._Width = _Width;
     }
 
-    constexpr void _On_precision(unsigned int _Precision) {
+    constexpr void _On_precision(int _Precision) {
         _Specs._Precision = _Precision;
     }
 
     constexpr void _On_type(_CharT _Type) {
-        _Specs._Type = static_cast<_CharT>(_Type);
+        _Specs._Type = static_cast<char>(_Type);

```

Comment:
We need to check if `_Type` can actually fit into a `char`, so that for example `\u0463` isn't interpreted as 'c'.

Reply:
Annoyingly we can't really do that in _Specs_checker without hacking around to get the character type.